### PR TITLE
PLNSRVCE-798 Update to latest JVM Build Service

### PR DESCRIPTION
**Pull request by stuartwdouglas**
Draft for now, as I seem to be having some issues testing this on my local KCP environment.

### CODE REVIEW

File: task/init/0.1/init.yaml [hunk 1]
```diff
@@ -16,9 +16,6 @@ spec:
     - name: rebuild
       description: Rebuild the image if exists
       default: "false"
-    - name: jvm-build-service-init
```
Comment (stuartwdouglas) [inline task/init/0.1/init.yaml:19]: We don't really have a solution for automatically initializing the workspace, until the new workspace setup controller is ready.

Comment (gabemontero) [reply task/init/0.1/init.yaml:19]: yep https://issues.redhat.com/browse/PLNSRVCE-763 is tracking that work

in the interim, manual initialization of the user workspace via a new for of the current jvm-build-service helm charts that do NOT install  the maven and git-clone tasks would cover this

**Comment by stuartwdouglas** [comment]
@Michkov @gabemontero This should be ready to go now, although I am still seeing some issues with JVM build service state changes running on KCP. I don't think it is related to this work though.

**Comment by gabemontero** [comment]
> @Michkov @gabemontero This should be ready to go now, although I am still seeing some issues with JVM build service state changes running on KCP. I don't think it is related to this work though.

@stuartwdouglas mentioned in slack he identified the issue ^^

that said, while of course the associated pipelinerun has been pruned on the staging cluster, the the yaml lint check failed

retesting now to see if it is related to these changes

/retest

**Comment by gabemontero** [comment]
ok looks like I can only retrigger from github actions

**Comment by gabemontero** [comment]
ok can't get it to re-run from github actions either 

closing / reopening

**Comment by gabemontero** [comment]
Yeah yaml lint had these complaints about your updates @stuartwdouglas 

```
step-lint-yaml-files
./pipelines/base/java-builder/patch.yaml
  29:72     error    no new line character at the end of file  (new-line-at-end-of-file)

./pipelines/base/docker-build/patch.yaml
  33:74     error    no new line character at the end of file  (new-line-at-end-of-file)

./.tekton/tasks/yaml-lint.yaml
  26:118    warning  too few spaces before comment  (comments)
  26:119    warning  missing starting space in comment  (comments)
```

**Comment by stuartwdouglas** [comment]
I have fixed the lint issues.

**Pull request merged**